vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporterController.java

```diff
@@ -194,9 +191,9 @@ private MongoCredentials getCredentials(String species, String dbName,
         return credentials;
     }
 
-    private CellbaseWSClient getCellbaseClient(String species, Properties evaProperties) throws URISyntaxException {
-        return new CellbaseWSClient(species, evaProperties.getProperty("cellbase.rest.url"),
-                                    evaProperties.getProperty("cellbase.version"));
+    private ChromosomeWsClient getChromosomeWsClient(String species, Properties evaProperties) throws URISyntaxException {
+        return new ChromosomeWsClient(species, evaProperties.getProperty("chromosome.rest.url"),
```

Comment:
If this was renamed to `EvaWsClient` it could be used for calls to any of the core API methods. The properties should be also renamed to `eva.rest.url` etc.